README.md

```diff
@@ -1,5 +1,7 @@
 # mention-bot
 
+[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy)
```

Comment:
The button is at the bottom, can you remove it here? I don't think that it should be that prominent